tests/com/jvm/external/jar/AddJarManifestEntryTest.java

```diff
@@ -0,0 +1,136 @@
+package com.jvm.external.jar;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.ByteStreams;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import rules.jvm.external.jar.AddJarManifestEntry;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.jar.Attributes;
+import java.util.jar.JarFile;
+import java.util.jar.JarOutputStream;
+import java.util.jar.Manifest;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import java.util.zip.ZipOutputStream;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class AddJarManifestEntryTest {
```

Comment:
Can you please add a test that a jar without a manifest and with "more than two entries" gets a manifest that that `JarInputStream` can find, please?

Reply:
There is a test for a jar without a manifest at https://github.com/bazelbuild/rules_jvm_external/blob/master/tests/unit/manifest_stamp/BUILD#L9  but it would be good to add a test for the `JarInputStream`